packages/compare-images/typescript/src/index-node-only.ts

```diff
@@ -18,3 +18,6 @@ export type { VectorMagnitudeNodeResult }
 
 import vectorMagnitudeNode from './vector-magnitude-node.js'
 export { vectorMagnitudeNode }
+
+import toScalarDouble from './to-scalar-double.js'
+export { toScalarDouble }
```

Comment:
Exporting `toScalarDouble` here for external use. This is useful to export test images as double and testing with native builds etc. May not be super crucial, but just for convenience.